device-types/Ciena/8110.yaml

```diff
@@ -0,0 +1,114 @@
+---
+manufacturer: Ciena
+model: '8110'
+slug: ciena-8110
+part_number: 170-8110-900
+u_height: 2
+is_full_depth: false
+weight: 8.16
+weight_unit: kg
+subdevice_role: parent
```

Comment:
Generally you do not need `subdevice_role` unless the device either slots into another (in which case it would be a child), or has `device-bays` to house child devices. If this device does not need this, please remove it